shell/content/docs/en-us/whats-new.md

```diff
@@ -1,7 +1,26 @@
+---
+title: What's New in 2.7
+---
+
+## New in 2.7.0
+
+### Rancher Extensions
+
+- You can now bring your own apps' UI and integrate it with the Rancher Dashboard to leverage a single UI. This relies on feature charts that would be manageable via a catalog. 
+
```

Comment:
@leflambeur Should we add something in here for support for OCI for helm for fleet?

Reply:
@deniseschannon I've grabbed top-level blurbs from a release notes draft, so the 2.7.0 material has more meat. At this point, I've not received much else from other teams but based on what's going into this version, this may be all.